ui/cypress/integration/volume/createvolume.spec.js

```diff
@@ -0,0 +1,70 @@
+beforeEach(() => {
+  cy.setupMocks();
+  cy.login();
+  cy.visit('/volumes/createVolume');
+});
+
+const VOLUME_NAME = 'test-volume-sparse';
+const STORAGECLASS = 'metalk8s';
+const VOLUME_LABEL_NAME = 'kubernetes.io/name';
+const VOLUME_LABEL_INVALID = 'kubernetes.io&/name';
+const VOLUME_LABEL_VALUE = 'test';
+const VOLUME_TYPE = 'sparseLoopDevice';
+const VOLUME_SIZE = '1 GiB';
+
+// interaction test for creating volume
+describe('the create button in create volume page', () => {
+  it('should be enabled and the Add button should be disabled when no labels is specified', () => {
+    cy.get('input[name=name]').type(VOLUME_NAME);
+
+    cy.get('.sc-select').eq(0).click();
+    cy.get('.sc-select__menu').eq(0).click();
+    cy.get('.sc-select').eq(1).click();
+    cy.get('.sc-select__menu')
+      .find(`[data-cy=storageClass-${STORAGECLASS}]`)
```

Comment:
We should select the element via the text presented to the user

Reply:
In the end, we got to keep the `data-cy`, the reason is:

>   /* we use `data-cy`is because the storageClass may display in both the select list and selected item, 
>      so we use unique selector to make sure to choose the item in the list. */